server/config.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const fs = require('fs');
+const dotenv = require('dotenv');
+
+this.windowEnv = function() {
+  // pull env key names from env-example
+  let exampleText = fs.readFileSync(`${__dirname}/../env-example`);
+  let keys = Object.keys(dotenv.parse(exampleText));
+
+  // safe parse .env
+  let env = {};
+  try {
+    let dottext = fs.readFileSync(`${__dirname}/../.env`);
+    env = dotenv.parse(dottext);
+  } catch (err) {}
+
+  // pull overrides out of process.env and .env
+  let envVar = {"sample": "works"};
+  for (let key of keys) {
+    if (process.env[key] !== undefined || (env[key] && env[key] !== '')) {
+      let val = process.env[key] || env[key];
+      if (['true', 'false', 'null', 'undefined'].indexOf(val) > -1) {
+        val = val;
+      } else if (isNaN(val) || val == '') {
```

Comment:
Close, but I think slightly off.  The previous code was attempting to manually json-encode these to write the ENV-string.  Here you're returning a plain javascript object with unencoded values.

For instance, if you had `LOGO_NAME=prx` in your `.env`, then for this loop you'd have `key = "LOGO_NAME";` and `val = "prx"`.  Which would result in quoting the value to `"'prx'"` on the next line.

I had this same problem in the current Publish codebase, and [solved it here](https://github.com/PRX/publish.prx.org/blob/master/lib/util.js#L11).  (Sorry for not remembering that earlier!)  A good rainy day project would be to encapsulate all these server/env-parsing scripts into a `prx-ng2-server` npm package, or something.  But we should probably wait till Play is on webpack to take that on.